plugins/chain_plugin/chain_plugin.cpp

```diff
@@ -1155,10 +1195,18 @@ void chain_plugin::plugin_initialize(const variables_map& options) {
             my->_account_query_db->commit_block(blk);
          }
          my->accepted_block_channel.publish( priority::high, blk );
+
```

Comment:
I don't think it matters, but for consistency lets publish to channel as the last thing in each of these.